docs/integration-guides/key-management.md

```diff
@@ -329,16 +329,91 @@ curl -d '{
     As a result of the command above, the nano\_node will return a signed, but not yet broadcasted transaction. Broadcasting of the signed transaction is covered in the [Broadcasting Transactions](#broadcasting-transactions) section.
 
 !!! info "Manually receiving first block"
-    The very first transaction on an account-chain, which is always a receive, is a bit special since it doesn't have a `"previous"` block. The process however, is very similar to a conventional receive transaction.
-
-    | Field          | Description |
-    |                |             |
-    | previous       | Value is 0 (32 0's) |
-    | account        | Same as normal receive. |
-    | representative | Choose a reliable, trustworthy representative. |
-    | balance        | Same as normal receive. This will be the transaction amount of the pairing send. |
-    | link           | Same as normal receive. |
-    | key            | Same as normal receive. |
+    The very first transaction on an account-chain, which is always a receive, is slightly special and deserves its own section [First Receive Transaction](#first-receive-transaction).
+
+---
+
+#### First Receive Transaction
+
+The first transaction of every account is crafted in a slightly different way. To open an account, you must have sent some funds to it with a [Send Transaction](#send-transaction) from another account. The funds will be **pending** on the receiving account. If you already know the hash of the pending transaction, you can skip Step 1.
+
+!!! example "Step 1: Obtain the pending transaction block hash"
+
+    Start with obtaining a list of pending transactions in your unopened account. Limit the response to the highest value transaction by using a combination of `sorting` and `count`.
+
+##### Request Example
+
+```bash
+curl -d '{
+  "action": "pending",
+  "account": "nano_1rawdji18mmcu9psd6h87qath4ta7iqfy8i4rqi89sfdwtbcxn57jm9k3q11",
+  "count": "1",
+  "sorting": "true",
+  "include_only_confirmed": "true"
+}' http://127.0.0.1:7076
+```
+
+##### Success Response
+
+```json
+{
+    "blocks": {
+        "5B2DA492506339C0459867AA1DA1E7EDAAC4344342FAB0848F43B46D248C8E99": "100"
+    }
+}
+```
+
+!!! example "Step 2: Build `block_create` request"
+    Using the block hash and raw transaction amount from the `pending` call response, along with other information, we can create the [`block_create`](/commands/rpc-protocol#block_create) RPC request. The only difference between the normal receive transactions is the `"previous"` field.
+
+    For more details on values, see the [Blocks Specifications](/integration-guides/the-basics/#blocks-specifications) documentation.
+
+    | Field              | Value |
+    |                    |       |
+    | `"type"`           | always the constant `"state"` |
+    | `"previous"`       | `"frontier"` from `account_info` response |
```

Comment:
Suggested update: `"frontier"` from `account_info` response, or `0` if first received transaction to open account

Reply:
Huh, weird I missed that. Should be "0" in all cases since this is request is specific to the first block, right?

Updated.